manifests/n/Notion/Notion/2.0.32/Notion.Notion.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Notion.Notion
+PackageVersion: 2.0.32
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: user
+InstallModes:
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://desktop-release.notion-static.com/Notion%20Setup%202.0.32.exe
+  InstallerSha256: 15136849EAFFC6BA1D308BD7BBDBD22CDA2DA5E4B6EE89FE68C3D782FB4B5B81
+ManifestType: installer
+ManifestVersion: 1.2.0
+ReleaseDate: 2022-08-11

```

Comment:
```suggestion
```
I don't see any release date mentioned for this version on the official website, so I think this optional field should be left out.